internal/soaktest/runner.go

```diff
@@ -80,6 +82,18 @@ func (runner *Runner) Run(ctx context.Context) error {
 		defer cancel()
 	}
 
+	if runner.config.RunDuration != 0 {
```

Comment:
Does this option conflicts with `RunForever`? What's the behaviour if both are set? Can we document this behaviour?